config/env.development

```diff
@@ -121,6 +121,17 @@ FEED_DISCOVERY_PORT=9999
 FEED_DISCOVERY_URL=http://localhost/v1/feed-discovery
 
 
+################################################################################
+# Parser Service
+################################################################################
+
+# Parser Service Port (default is 8888)
+PARSER_PORT=8888
+
+# Parser Service URL
+PARSER_URL=http://localhost/v1/parser
```

Comment:
http vs. https

Reply:
All the other services in the file seem to be using http for local development

Reply:
Yeah, it should be http.